pyluos/modules/color.py

```diff
@@ -5,6 +5,7 @@
 class Color(Module):
     def __init__(self, id, alias, robot):
         Module.__init__(self, 'Color', id, alias, robot)
+        self._time = 0.2
```

Comment:
At module boot, `self._time` is 0. But when pyluos connect to it we don't know the current value.
I suggest to put `self._time` to `None`